x-pack/plugin/core/src/test/resources/rest-api-spec/schema/transform_time_retention_policy_config.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/elastic/elasticsearch/master/x-pack/plugin/core/src/test/resources/rest-api-spec/schema/transform_time_retention_policy_config.schema.json",
+  "description": "schema definition for a retention policy",
+  "additionalProperties": false,
+  "title": "Root",
+  "type": "object",
+  "required": [
+    "field",
+    "max_age"
+  ],
+  "properties": {
+    "field": {
+      "$id": "#root/retention_policy/time/field",
+      "title": "Field",
+      "type": "string",
+      "examples": [
+        "@timestamp"
+      ],
+      "pattern": "^.*$"
+    },
+    "max_age": {
+      "$id": "#root/retention_policy/time/max_age",
+      "title": "Max age",
+      "type": "string",
+      "default": "60s",
+      "examples": [
+        "10s"
+      ],
+      "pattern": "^.*$"
```

Comment:
I think this pattern is incorrect. This is a timevalue field, so it will always start with some digits and end with a number. 

But, again, I am not sure how particular we want the specs to be.